### ISSUE TITLE
Improve docs

### ISSUE THREAD

**Issue by lsdrfrx** [norse/norse#374]
Please, make a better docs for STDP and Izhikevich modules with examples, because it's not clear how to use them

**Comment by Jegp** [comment]
Thanks for the suggestion @lsdrfrx. I agree that the documentation of the modules could be better.

I'm tied up with other things, I'm afraid. But better documentation and contributions are more than welcome! And I would be happy to either have a chat about how we can improve it together or provide feedback on a PR.

**Comment by lsdrfrx** [comment]
Afternoon, @Jegp
It would be great if I could help with docs. Can I contact the developer who implemented these modules to clarify how they work?

**Comment by Jegp** [comment]
Happy to hear that :-)

I've been involved with the Izhikevich, although the lead developer was @adelpierre  - maybe he has time to help you out? 

@cpehle implemented the STDP functionals. I don't know about his availability, though. I will mention, though, that I think there may be better ways to go about it. If you're interested in driving this forward, I think it could be valuable to refactor the code slightly and allow for other learning rules like STDP.

**Comment by adelpierre** [comment]
Hey @lsdrfrx and Jens !

I'll be happy to help with any questions you have, though I might need a bit of time to get back in the saddle as I have not worked on this code for a while.

**Comment by Jegp** [comment]
That's great to hear! Could you put a few words on which kind of doc / what information would be helpful to have in the docs? Usage? Examples?

**Comment by adelpierre** [comment]
I could try to create a example notebook with the Izhikevich neuron model, if that sounds good I can see if I can do it during the weekend or next week

**Comment by Jegp** [comment]
That sounds pretty neat. My advice would be to keep it simple, and to check with @lsdrfrx what use case they're looking for.

But a tutorial/notebook on the Izhikevich model would be good to have, I think.

**Comment by lsdrfrx** [comment]
@adelpierre 
Yes, example notebook with Izhikevich neuron model and STDP would be great!
In my project, the implementation of the Izhikevich neuron model using STDP is very necessary. Due to incomplete documentation, it was not possible to figure out how to implement this idea.
To prevent other users from encountering this problem, I would like to supplement the documentation.
I want to thank you for this great work! The framework is really good :)

**Comment by lsdrfrx** [comment]
In the laboratory, we study how neuromorphic neural networks can be used in the tasks of neuroscience, physics and chemistry. At the moment, I just need to train a neural network to classify handwritten digits on MNIST.
By the way, now I am personally working on creating a simulator of biological neural networks that implements various models of spike neurons. At the moment I am developing the Izhikevich neuron.
The topic of neuromorphic networks attracts me very much because it is not as boring as the classic ANN :D

**Comment by Jegp** [comment]
Thanks for the input and kind words :-) Happy to hear it's useful!

Your work sounds interesting. I would be curious to hear more about your neuron simulator. Are you on our discord? Maybe it's easier to continue the conversation there. https://discord.gg/JXJhJRub

**Comment by AakiraOtok** [comment]
@Jegp The link seems to be dead. I really, really want to join the Norse community. Could you please provide another link? 😊

**Comment by Jegp** [comment]
Happy to hear that! Does this link work?
https://discord.gg/7fGN359

**Comment by AakiraOtok** [comment]
It worked! Thanks<3
